inst/tutorials/data_visualization_basic/data_visualization_basic.Rmd

```diff
@@ -0,0 +1,301 @@
+---
+title: "Introduction to data visualization"
+author: "Julia Anstett and Dr. Stephan Koenig (adpated from Dr. Kim Dill-McFarland)"
+date: "version `r format(Sys.time(), '%B %d, %Y')`"
+output:
+  learnr::tutorial:
+    progressive: true
+    allow_skip: true
+runtime: shiny_prerendered
+description: Basic data visualization with ggplot2 focusing on dot plots.
+---
+
+```{r setup, include = FALSE}
+# General learnr setup
+library(learnr)
+knitr::opts_chunk$set(echo = TRUE)
+library(educer)
+# Helper function to set path to images to "/images" etc.
+setup_resources()
+
+library(dplyr)
+library(readr)
+
+raw_dat <- geochemicals
+
+dat <- mutate(raw_dat, Depth = as.factor(Depth),
+              Cruise = as.factor(Cruise))
+
+dat <- filter(dat, Depth %in% c(10, 100, 200),
+              # The cruises that took place in Februrary can be determined
+              # using the Date variable and functions that we won't discuss in
+              # these data science modules
+              Cruise %in% c(18, 30, 42, 54, 66, 80, 92))
+```
+
+## Introduction
+This tutorial provides guided, hands-on instruction to create and modify plots using the `ggplot2` package in R. It also includes plots of an ANOVA analysis.
+
+After this tutorial, you will be able to:
+
+* Create dot and box plots in `ggplot2`
+* Modify attributes of ggplots
+* Complete and interpret the output of ANOVAs in R
+
+## Setup
+Prior to starting this tutorial, please complete the *Pre-module download assignment* to obtain all the necessary software and data.
```

Comment:
Is this tutorial a part of a larger curriculum? Where do students go to access the pre-module download assignment?